install_config/aggregate_logging.adoc

```diff
@@ -814,13 +814,13 @@ to a value of `1` or higher to create replicas.
 
 Use the following scenarios as a guide for an {product-title} cluster with three Elasticsearch nodes:
 
-* If you can tolerate one Elasticsearch node going down,
-set `openshift_logging_es_number_of_replicas` to `1`. This ensures
-that two nodes have a copy of all of the Elasticsearch data in the cluster.
+* With `openshift_logging_es_number_of_replicas` set to `1`, two nodes have a copy of all of the Elasticsearch data in the cluster.
+This ensures that if a node with the Elasticsearch data goes down, another node has a copy of all of the Elasticsearch data in the cluster.  
```

Comment:
Probably fine, but just wondering if "if a node with the Elasticsearch data" should be "if a node with Elasticsearch data"